homeassistant/components/plugwise/entity.py

```diff
@@ -42,17 +42,17 @@ def __init__(
             configuration_url=configuration_url,
             identifiers={(DOMAIN, device_id)},
             connections=connections,
-            manufacturer=data.get("vendor"),
-            model=data.get("model"),
+            manufacturer=data.get(CONF_VENDOR),
```

Comment:
This is not configuration? Why does it use `CONF_*` constant?

Reply:
I copied this from another integration, don't remember which one.

Reply:
Would you prefer I revert or change to another varname in CAPITALS?

Reply:
I'll revert these changes.